packages/client/tests/functional/create-many/_matrix.ts

```diff
@@ -0,0 +1,26 @@
+import { defineMatrix } from '../_utils/defineMatrix'
+
+export default defineMatrix(() => [
+  [
+    {
+      provider: 'mysql',
+      providerFeatures: '',
+    },
+    {
+      provider: 'postgresql',
+      providerFeatures: '',
+    },
+    {
+      provider: 'mongodb',
+      providerFeatures: '',
+    },
+  ],
+  [
```

Comment:
Do those previews features make any difference for this particular test?

Reply:
No, my bad a copy and paste error!